can_bus/quadruna/VC/VC_tx.json

```diff
@@ -717,5 +717,18 @@
                 "unit": "L/H"
             }
         }
+    },
+    "StateOfCharge": {
+        "msg_id": 226,
+        "cycle_time": 100,
```

Comment:
See the other comment about moving broadcast to 1Hz, maybe up cycle time to 1000 if that change is performed

Reply:
fixed!